platform/android/export/export_plugin.cpp

```diff
@@ -1056,6 +1057,15 @@ void EditorExportPlatformAndroid::_fix_manifest(const Ref<EditorExportPreset> &p
 					Vector<bool> feature_required_list;
 					Vector<int> feature_versions;
 
+					String current_renderer = GLOBAL_GET("rendering/renderer/rendering_method.mobile");
+					bool has_vulkan = current_renderer == "forward_plus" || current_renderer == "mobile";
+					if (has_vulkan) {
```

Comment:
I think the better check would be to query `rendering/rendering_device/driver.android`, which would be `"vulkan"`.

Currently that's the only available driver for RenderingDevice-based rendering methods (forward_plus and mobile), but in theory there could be other implementations in the future (e.g. a Metal implementation for macOS/iOS, and Direct3D 12 for Windows). For Android, it's unlikely to happen, but who knows, maybe there will be some OpenGL ES 3.2-based high end RenderingDevice API in the future. Or WebGPU-native or something.

Reply:
Though actually, you'd need to check both settings probably - `rendering/rendering_device/driver.android` is always defined, but only relevant *if* the selected rendering method is a RenderingDevice-based one... and I don't think there's an easy way to know that aside from checking whether it's `"forward_plus"` or `"mobile"`.

So the "safest" way might be:
```suggestion
					String current_renderer = GLOBAL_GET("rendering/renderer/rendering_method.mobile");
					bool has_vulkan = (current_renderer == "forward_plus" || current_renderer == "mobile") && GLOBAL_GET("rendering/rendering_device/driver.android") == "vulkan";
					if (has_vulkan) {
```

Also relevant for these checks: #71542